pytensor/tensor/blas.py

```diff
@@ -146,18 +133,18 @@ def check_init_y():
     if check_init_y._result is None:
         if not have_fblas:
             check_init_y._result = False
-
-        y = float("NaN") * np.ones((2,))
-        x = np.ones((2,))
-        A = np.ones((2, 2))
-        gemv = _blas_gemv_fns[y.dtype]
-        gemv(1.0, A.T, x, 0.0, y, overwrite_y=True, trans=True)
-        check_init_y._result = np.isnan(y).any()
+        else:
+            y = float("NaN") * np.ones((2,))
+            x = np.ones((2,))
+            A = np.ones((2, 2))
+            gemv = get_blas_funcs(names="gemv", dtype=y.dtype)
```

Comment:
I suppose we want to cache this so we are not calling it every iteration of perform. Elemwise does something like that for the respective ufunc

Otherwise does cloudpickle work if we were using this in the old dict instead?